afdd_stack/openfdd_stack/tests/platform/test_bootstrap_modes.py

```
@@ -81,3 +83,83 @@ def test_bootstrap_rejects_unknown_option():
+def test_volttron_duplicate_ssl_quarantine_moves_config(tmp_path):
+    cfg = tmp_path / "config"
+    cfg.write_text(
+        "[volttron]\n"
+        "web-ssl-cert = /a.pem\n"
+        "web-ssl-cert = /b.pem\n"
+        "web-ssl-key = /k.key\n",
+        encoding="utf-8",
+    )
+    script = _duplicate_ssl_quarantine_function_source()
+    script += f'run_volttron_quarantine_duplicate_ssl_config "{cfg}"\n'
+    res = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)